sormas-backend/src/main/java/de/symeda/sormas/backend/contact/ContactFacadeEjb.java

```diff
@@ -767,10 +771,14 @@ public long countMaximumFollowUpDays(ContactCriteria contactCriteria) {
 		final CriteriaBuilder cb = em.getCriteriaBuilder();
 		final CriteriaQuery<Long> cq = cb.createQuery(Long.class);
 		final Root<Contact> contactRoot = cq.from(Contact.class);
-		final ContactJoins joins = new ContactJoins(contactRoot);
+
+		final ContactQueryContext contactQueryContext = new ContactQueryContext(cb, cq, contactRoot);
+
+		final ContactJoins joins = (ContactJoins) contactQueryContext.getJoins();
+		// fixme - why was this added?
```

Comment:
To make sure that visits are loaded because I think they're lazy; remove this comment